integrated_channel_plugin_sapsf/models.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from model_utils.models import TimeStampedModel, TimeFramedModel
+
+
+# Create your models here.
+
+class OCNWebServicesConfiguration(TimeStampedModel):
+    """
+    The Global configuration for integrating with SuccessFactors.
+    """
+
+    completion_status_api_path = models.CharField(max_length=255)
+    course_api_path = models.CharField(max_length=255)
+    oauth_api_path = models.CharField(max_length=255)
+    provider_id = models.CharField(max_length=100, default='EDX')
+
+    class Meta:
+        app_label = 'integrated_channel_plugin_sapsf'
+
+
+class OCNWebServicesEnterpriseCustomerConfiguration(TimeStampedModel):
+    """
+    The Enterprise specific configuration we need for integrating with SuccessFactors.
+    """
+
+    enterprise_customer_uuid = models.UUIDField(unique=True)
+    sapsf_base_url = models.CharField(max_length=255)
+    key = models.TextField(blank=True, verbose_name="Client ID")
+    secret = models.TextField(blank=True, verbose_name="Client Secret")
```

Comment:
How long are the keys and secrets? I would think that `CharField` is a better fit for that type of data. (Do you want a textarea or a single-line input in the django admin?)